linkerd/protocol/http/src/main/scala/io/buoyant/linkerd/protocol/http/ResponseClassifiers.scala

```diff
@@ -45,6 +47,11 @@ object ResponseClassifiers {
       Method.Put,
       Method.Delete
     ))
+
+    val Write = Idempotent.withMethods(Set(
```

Comment:
> This classifier marks ONLY writes (POST and PUT) as retryable but all other methods as non-retryable. Is this what you want? Would it be more useful to have a response classifier which marks ALL methods as retryable?

@adleong I thought the because we define `Write` based on `Idempotent.withMethods` it would take the verbs from `Idempoent` as well i.e.  put, delete, get, head, options, trace

Reply:
if that is the case maybe a better name would be `All` and call the response classifier `retryableAll5XX` ? Our intention was to retry regardless of the verb with this PR

Reply:
Ah, you're right, I got mixed up because of the name.  `retryableAll5XX` sounds good to me.